MainPresenter.py

```diff
@@ -0,0 +1,5 @@
+
+class MainPresenter():
+    def __init__(self,MainView):
```

Comment:
Good. I suggest keeping the `MainView` argument as lower case i.e. `main_view` . You are passing an object of type `MainView` not the class.